### PR TITLE
Update adtelligent ortb endpoint

### PR DESCRIPTION
**Pull request by GeneGenie**
Update ortb endpoint for Adtelligent bidder
Extend Adtelligent csync url to accept gdpr and usp  consents

### CODE REVIEW

File: adapters/adtelligent/adtelligent.go [hunk 1]
```diff
@@ -55,7 +55,9 @@ func (a *AdtelligentAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *
 
 	imps := request.Imp
 	request.Imp = make([]openrtb.Imp, 0, len(imps))
-
+	if request.Device != nil && request.Test == 1 {
+		request.Device.IP = getIP(request.Device.IP)
```
Comment (SyntaxNode) [inline adapters/adtelligent/adtelligent.go:59]: Please clone the Device first before changing the IP. This is shared memory among adapters.

File: config/config.go [hunk 2]
```diff
@@ -498,7 +498,7 @@ func (cfg *Configuration) setDerivedDefaults() {
...
 	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderAdkernelAdn, "https://tag.adkernel.com/syncr?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3DadkernelAdn%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%7BUID%7D")
 	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderAdpone, "https://usersync.adpone.com/csync?redir="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dadpone%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%7Buid%7D")
-	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderAdtelligent, "https://sync.adtelligent.com/csync?t=p&ep=0&redir="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dadtelligent%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%7Buid%7D")
+	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderAdtelligent, "https://sync.adtelligent.com/csync?t=p&ep=0&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dadtelligent%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%7Buid%7D")
```
Comment (SyntaxNode) [inline config/config.go:501]: Please update the syncer test with the new url.

File: adapters/adtelligent/adtelligent.go (hunk 1 above)
Comment (GeneGenie) [reply adapters/adtelligent/adtelligent.go:59]: better to delete this, and even move to core ?! what is the reason for sending ::1 to exchanges ?

File: config/config.go (hunk 2 above)
Comment (GeneGenie) [reply config/config.go:501]: done

File: adapters/adtelligent/adtelligent.go (hunk 1 above)
Comment (SyntaxNode) [reply adapters/adtelligent/adtelligent.go]: Sure. That sounds good to me.

**Pull request merged**